### DISCUSSION TITLE
OPA rego policy - URL dynamic path parameters glob.match

### DISCUSSION THREAD

**Discussion by dsnr-coder** [open-policy-agent/community#535] [OPA and Rego]
Hi Everyone,

I'm using OPA to write an access policy to a microservice, and I'm now tackling the problem of matching an URL containing a path param.

The URL follows the pattern: /test-service/{uuid}/users/{uuid1}/groups/

I tried with many ways nothing has worked in my favour.
	glob.match("/test-service/{}/users/{}/groups/", [], input.path)

Please find the rego URL below that I have tried.
https://play.openpolicyagent.org/p/82me37Kawh

Any help would be greatly appreciated

**Comment by anderseknert** [comment]
Here's an example: https://play.openpolicyagent.org/p/rrjCoIZikf

One thing to be aware of is that a trailing `/` or not is going to be significant here, I think.. so you might want to "normalize" that in the input before matching to make sure it's added (or removed depending on your preference).
